Lib/argparse.py

```diff
@@ -832,9 +833,53 @@ def _get_kwargs(self):
         ]
         return [(name, getattr(self, name)) for name in names]
 
+    def format_usage(self):
+        return self.option_strings[0]
+
     def __call__(self, parser, namespace, values, option_string=None):
         raise NotImplementedError(_('.__call__() not defined'))
 
+class BooleanOptionalAction(Action):
+    def __init__(self,
+                 option_strings,
+                 dest,
+                 const=None,
```

Comment:
Is it intentional that we don't pass `const` to super? If so, maybe we shouldn't accept it at all.

Reply:
This is certainly suspect. I don't remember, I will have a look tomorrow.

Reply:
Thanks @hauntsaninja, I opened https://bugs.python.org/issue40862 and https://github.com/python/cpython/pull/20623 to fix this issue.